interface/forms/prior_auth/FormPriorAuth.class.php

```diff
@@ -52,17 +54,11 @@ function FormPriorAuth($id = "", $_prefix = "")
         }
     }
 
-    function toString($html = false)
+    function __toString()
```

Comment:
In this case, since it already exists (and is not being called in the codebase), then ok to fix and change it to __toString(note it is not correct since it doesn't list all variables, but is not worth the time/resources to add this).